buildSrc/src/main/java/com/google/firebase/gradle/plugins/DackkaPlugin.kt

```diff
@@ -206,11 +206,20 @@ abstract class DackkaPlugin : Plugin<Project> {
             include("**/package-list")
         }.toList()
 
+    // TODO(b/243534168): Remove when fixed
+    private fun projectSpecificSources(project: Project) =
+        when (project.name) {
+            "firebase-common" -> {
+                project.project(":firebase-firestore").files("src/main/java/com/google/firebase").toList()
```

Comment:
Isn't "files()" recursive in this case? i.e. won't it include all of firestore's source as opposed to just `Timestamp.java`?

Reply:
Actually, it includes the directory. Dackka doesn't support individual files, only directories. As a work around, we suppress everything but timestamp for common in `projectSpecificSuppressedFiles`